marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/impl/entity-lib.mjs

```diff
@@ -26,17 +24,17 @@ import consts from "/data-hub/5/impl/consts.mjs";
 import httpUtils from "/data-hub/5/impl/http-utils.mjs";
 import hubUtils from "/data-hub/5/impl/hub-utils.mjs";
 
-const sem = require("/MarkLogic/semantics.xqy");
```

Comment:
Don't we need sem to be defined?

Reply:
There are some functions that need imported and others that don't because they are defined at the C++ layer.